tests/test_join.py

```diff
@@ -60,7 +61,7 @@ def test_join_both_empty_targets(db: gp.Database, t1: gp.Table, t2: gp.Table):
     assert list(list(ret)[0].keys()) == ["id1", "idd1", "n1", "id2", "idd2", "n2"]
 
 
-def test_join_all_emp_targets(db: gp.Database, t1: gp.Table, t2: gp.Table):
+def test_join_all_empty_targets(db: gp.Database, t1: gp.Table, t2: gp.Table):
```

Comment:
I still don't get why the test is named as `xxx_empty_xxx` here, what does the empty mean here?

Reply:
> I still don't get why the test is named as `xxx_empty_xxx` here, what does the empty mean here?

It means for example we want to select all columns of table t1 and 0 columns of table t2. Well is not very meaningful in a join case but added this test case.